components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2605,17 +2599,14 @@ def getGroupsMemberOf(self):
         :rtype:         :class:`ExperimenterGroupWrapper` generator
         """
 
-        q = self.getQueryService()
-        p = omero.sys.Parameters()
-        p.map = {}
-        p.map["ids"] = rlist([rlong(a)
-                              for a in self.getEventContext().memberOfGroups])
-        sql = "select e from ExperimenterGroup as e where e.id in (:ids)"
-        for e in q.findAllByQuery(sql, p, self.SERVICE_OPTS):
-            if e.name.val == "user":
```

Comment:
@chris-allan @will-moore do you remember if there were any reason why we added exclusion for user group in ```getGroupsMemberOf``` but not in ```getGroupsLeaderOf```? Does it make sense if I unify it? Do you have any preferences?